include/json-helper.h

```diff
@@ -59,6 +59,14 @@ gint64 json_get_int(JsonNode *json_node, const gchar *path, GError **error);
  */
 JsonArray* json_get_array(JsonNode *json_node, const gchar *path,
                 GError **error);
+
+/**
+ * @brief Check if the given path matches an element in json_node.
+ *
+ * @param[in]  json_node JsonNode to query
+ * @param[in]  path      Query path
```

Comment:
This input here in detail would be a "JSONPath expression". glib methods call this variable "expression"

Reply:
Right, fixed.